### DISCUSSION TITLE
Possibility to add links to field labels

### DISCUSSION THREAD

**Discussion by ovenum** [tobimori/kirby-dreamform#57] [Feature Requests]
Being able to add links to checkbox and radio field labels can be useful when more context for an option is required. 
A use case for this can be linking to the Dataprotection statement that has to be accepted before a form is submitted. 

![image](https://github.com/tobimori/kirby-dreamform/assets/47149622/bbdf96cb-ad3a-4204-b3d7-f31c9075e586)

**Comment by tobimori** [comment]
This will be included in the next minor release
